jupyter_sphinx/ast.py

```diff
@@ -152,6 +163,136 @@ def run(self):
         cell_node += cell_input
         return [cell_node]
 
+class InputCell(Directive):
```

Comment:
I think `CellInput` (and likewise `CellOutput`) are more appropriate names because both refer to an input and an output *of a cell*.